CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 0.7.1 (Next)
+### 0.8.1 (Next)
```

Comment:
ok, I went over the guidelines and because this is a breaking change I bumped to 0.8, that was why. I'll put it back